mobile/src/features/Swap/useCases/ListOrders/ListOrders.tsx

```diff
@@ -51,8 +53,25 @@ export const ListOrders = () => {
     onBack: navigateToTxHistory,
   })
 
+  const isTestnet = network !== Chain.Network.Mainnet
+
+  if (isTestnet) {
+    return (
+      <View style={[a.flex_1, a.gap_lg, a.justify_center, a.align_center]}>
+        <Text style={[a.heading_3_medium, {color: p.text_gray_medium}]}>
+          Swap orders are not available on testnet
```

Comment:
please create i18n keys, this screen is also shown in production after network switcher feature

Reply:
Isn't the whole swap feature gated to prod? There's a screen for swap not working on testnet.

Reply:
I think now there is an icon to navigate to orders that was moved to the top right corner